hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CommonConfigurationKeys.java

```diff
@@ -243,6 +243,9 @@ public class CommonConfigurationKeys extends CommonConfigurationKeysPublic {
       "hadoop.security.token.service.use_ip";
   public static final boolean HADOOP_SECURITY_TOKEN_SERVICE_USE_IP_DEFAULT =
       true;
+  public static final String HADOOP_SECURITY_HOSTNAME_CACHE_INTERVAL =
```

Comment:
Is it better to replace ".interval" with ".secs" to specify the expected time-unit ?

Reply:
Good suggestion! 
Thanks for your feedback!
I look through interval related configuration keys in the hadoop project, and EXPIRE_INTERVAL_SECS(expire-interval.secs) maybe better.